sdk-actors/src/main/java/io/dapr/actors/client/DaprChannel.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.client;
+
+import io.dapr.client.DaprApiProtocol;
+import io.dapr.client.DaprHttpBuilder;
+import io.dapr.config.Properties;
+import io.dapr.v1.DaprGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+/**
+ * Holds a channel for Dapr sidecar communication. Channel should be reused.
+ */
+public class DaprChannel implements AutoCloseable {
```

Comment:
Would a better name for this be `ActorChannel`? or something similar since it's scoped to actors not dapr as a whole.

Reply:
It is scoped for Actors because it is in the `io.dapr.actors.client` package.

Reply:
We have the same for the `DaprClient` interface where we have on in `io.dapr.actors.client` and one in `io.dapr.client` and they are different.

Reply:
Since it now implements the `DaprClient` interface, I decided to name it `ActorClient` similar to how we name `ActorProxy`. So, it should resolve this ambiguity now.